MIGRATION.md

```diff
@@ -4181,7 +4181,7 @@ We're sorry if this inconveniences you, we have tried via this document and prov
 
 Storybook will now use webpack 2 (and only webpack 2).
```

Comment:
webpack 2 or 5?

Reply:
@kbrilla, thanks for the comment and your question. Appreciate it 🙏 . Actually, it's Webpack 2, this pull request and change is referring to Storybook version 3.x which was released a long time ago.

Reply:
`migrate/5` took me by surprise here - thank You for clarification